_tuning-your-cluster/availability-and-recovery/workload-management/wlm-feature-overview.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: default
+title: Workload management
+nav_order: 70
+has_children: true
+parent: Availability and recovery
+---
+
+Introduced 2.18
+{: .label .label-purple }
+
+# Workload management
+
+Workload management allows users to group search traffic and isolate network resources, preventing the overuse of network resources by specific requests. It offers the following benefits:
+
+- Tenant-level admission control and reactive query management. When resource usage exceeds configured limits, it automatically identifies and cancels demanding queries, ensuring fair resource distribution.
+
+- Tenant-level isolation within the cluster for search workloads, operating at a node level.
+
+## Installing workload management
+
+To install workload management, use the following command: 
+
+```json
+./bin/opensearch-plugin install workload-management
+```
+{% include copy-curl.html %}
+
+## Permissions
+
+Only users with administator-level permissions can use workload management. 
+
+## Query groups
+
+A _query group_ is a logical group of tasks with defined resource limits. System administrators can dynamically manage query groups using the Workload management APIs. These query groups can be used to make search requests with resource limits. 
```

Comment:
> Workload management APIs

It should be workload management REST APIs.

Reply:
When referring to API names, we always capitalize the first letter. See the API style guide, https://github.com/opensearch-project/documentation-website/blob/main/API_STYLE_GUIDE.md#api-name